sequencer/closingsignalsmanager.go

```diff
@@ -55,13 +57,25 @@ func (c *closingSignalsManager) checkGERUpdate() {
 	for {
 		time.Sleep(c.cfg.ClosingSignalsManagerWaitForCheckingGER.Duration)
 
-		ger, _, err := c.dbManager.GetLatestGer(c.ctx, c.cfg.GERFinalityNumberOfBlocks)
+		lastL1BlockNumber, err := c.etherman.GetLatestBlockNumber(c.ctx)
+		if err != nil {
+			log.Errorf("error checking GER update: %v", err)
+			continue
```

Comment:
It would be good to change this message avoid confusing it with line 73

Reply:
Changed